termtyper/ui/widgets/option.py

```diff
@@ -40,7 +41,11 @@ def __init__(
 
     def reset_cursor(self):
         if self.section:
-            self._cursor = self.options.index(parser.get(self.section, self.name))
+            try:
+                self._cursor = self.options.index(parser.get(self.section, self.name))
+            except NoOptionError:
+                parser.add_default_config(self.section, self.name)
+                self.reset_cursor()
```

Comment:
I don't think that it is a good idea to handle `NoOptionError` here.
It's the `Parser` class' job to internally handle these errors